docs/roman/ramp_fitting/description.rst

```diff
@@ -156,64 +118,80 @@ sufficient; they are given as:
 | 100               |                        | 10       |
 +-------------------+------------------------+----------+
 
-Segment-specific Computations:
-------------------------------
-The variance of the slope of a segment due to read noise is:
+Segment-Specific Computations
+-----------------------------
+
+The segment fitting implementation is based on Section 5 of Casertano et.
+al. 2022. A set of auxiliary quantities are computed as follows:
 
 .. math::
-   var^R_{s} = \frac{12 \ R^2 }{ (ngroups_{s}^3 - ngroups_{s})(group_time^2) } \,,
+   F0 &= \sum_{i=0}^{N-1} W_i
+
+   F1 &= \sum_{i=0}^{N-1} W_i \bar t_i
 
-where :math:`R` is the noise in the difference between 2 frames,
-:math:`ngroups_{s}` is the number of groups in the segment, and :math:`group_time` is the group
-time in seconds (from the exposure.group_time).
+   F2 &= \sum_{i=0}^{N-1} W_i \bar t_i^2
 
-The variance of the slope in a segment due to Poisson noise is:
+The denominator, :math:`D`, is calculated as a single two-dimensional array:
 
 .. math::
-   var^P_{s} = \frac{ slope_{est} }{  tgroup \times gain\ (ngroups_{s} -1)}  \,,
+   D = F2 \cdot F0 - F1^2
 
-where :math:`gain` is the gain for the pixel (from the GAIN reference file),
-in e/DN. The :math:`slope_{est}` is an overall estimated slope of the pixel,
-calculated by taking the median of the first differences of the groups that are
-unaffected by saturation and cosmic rays. This is a more
-robust estimate of the slope than the segment-specific slope, which may be noisy
-for short segments.
 
-The combined variance of the slope of a segment is the sum of the variances:
+The resultant coefficients, :math:`K_i`, are computed as N two dimensional
+arrays:
 
 .. math::
-   var^C_{s} = var^R_{s} + var^P_{s}
+   K_i = (F0 \cdot \bar t_i - F1) \cdot W_i / D
 
+The estimated slope, :math:`\hat F`, is computed as a sum over the resultants
+:math:`R_i` and the coefficients :math:`K_i`:
 
-Exposure-level computations:
-----------------------------
+.. math::
+   \hat F = \sum_{i} K_i R_i
+
+The calculation is skipped for pixels that have :math:`D = 0`. Note that the coefficient
+:math:`K_i` vanishes for each resultant that has a bad pixel, as a consequence of :math:`W_i`
+vanishing.
```

Comment:
I don't really think of this in this way.  The W_i are what you specify above and never vanish.  But bad entries in the DQ array are used to split the ramps into segments, and within each segment, a separate fit is done, where all of the W_i are nominal.